docs: Docs for v0.17.0 🚀 (#7248)
excalidraw/excalidraw: dev-docs/docs/@excalidraw/excalidraw/api/excalidraw-element-skeleton.mdx

```

![image](https://github.com/excalidraw/excalidraw/assets/11256141/a8b047c8-2eed-4aea-82a2-e1e6bbddb8d4)

### Frames

> To start creating frames programmatically, you need to pass `type`, `children` and rest of attributes are optional.

Reviewer: ```suggestion
To create a frame, you need to pass `type`, `children` (Excalidraw elements). The rest of the attributes are optional.
```
